components/lib/crash/build.gradle

```diff
@@ -33,6 +37,8 @@ dependencies {
 
     implementation project(':support-base')
 
+    implementation project(':service-glean')
```

Comment:
I wonder how we can prevent having a hard-dependency on Glean for every user of this library. I can think of two options:
* 1) We reverse the dependency and GleanService lives in Glean. However then Glean has a dependency on lib-crash - does this make more sense?
* 2) We turn this dependency into a compileOnly dependency like we did with Sentry below: An app using Glean will already depend on it and therefore the dependency gets resolved - but for everyone else it does not and they do not need to pay the cost.

Reply:
The other way I was considering going with this was to implement the Glean CrashReporterService in the application rather than the library, but I'll definitely look into doing in the service-glean.  We really didn't want a hard dependency on the lib-crash either so I understand not wanting the hard dependency the other way.

Reply:
Can we try instead to move on with option (2)?

Reply:
Yes, absolutely!  I meant to say that was the route I would attempt at first, but I guess I didn't finish my thought in the previous comment.